src/vstart.sh

```diff
@@ -1130,6 +1134,16 @@ EOF
     fi
 }
 
+start_exporter() {
+    echo "Starting exporter daemon..."
+
+    # Define socket directory for the exporter
+    # Start the exporter daemon 
+    prun $CEPH_BIN/ceph-exporter \
```

Comment:
Probably it is needed to add the parameter to point the ceph config file (like other daemons):
`-c "$conf_fn"`

Reply:
It should probably respect `-i <ip>` and `-l` options that `vstart.sh` accepts as well, utilizing `--addrs` option of the daemon (unless I'm misinterpreting what it does).